configs/public_rpcs.go

```diff
@@ -41,3 +41,18 @@ var networkRPCs = map[string]RPC{
 		},
 	},
 }
+
+var networkWSs = map[string]RPC{
```

Comment:
I think this should be integrated as part of RPC struct, same as PublicRPCs, Like PublicWSS

Reply:
@AntiD2ta

Reply:
This could be a change without any harm, but let's always be wary of changing code in minor version releases.

This is following the **Open-Closed Principle** from the SOLID principles of object-oriented design. The Open-Closed Principle states that:

"Software entities (classes, modules, functions, etc.) should be open for extension, but closed for modification."

This means that you should design your modules so that they can be extended to accommodate new functionalities without altering existing code. By doing so, you reduce the risk of introducing bugs into established, working code and make your system more maintainable and scalable.

Key Points:

**Open for Extension**: You can add new features or behaviors to the module.
**Closed for Modification**: Existing code remains unchanged, ensuring stability.

I would put a TODO to consider it in a further refactor.

Reply:
This could be a change without any harm.

I would put a TODO to consider it in a further refactor if @khalifaa55 doesn't have buffer to address the proposal.

Reply:
[Done](https://github.com/NethermindEth/sedge/pull/411/commits/2ecb0644c4a73fe9828e91923554594d07f471e6)